.travis.yml

```diff
@@ -9,6 +8,9 @@ matrix:
     - os: linux
       env: DOCKERIMAGE=linux-anvil-comp7
 
+    - os: linux
+      env: DOCKERIMAGE=linux-anvil-ppc64le
```

Comment:
Need a better name.

Reply:
Like?

Reply:
What's the difference between linux-anvil2 and linux-anvil-ppc64le ?

Reply:
Now nothing aside from what they get built on.